pandas/core/frame.py

```diff
@@ -1078,6 +1078,9 @@ def to_string(
         %(shared_params)s
         line_width : int, optional
             Width to wrap a line in characters.
+        min_rows : int, optional
```

Comment:
why did you move the parmeter?

Reply:
The parameter `min_rows` used to be documented in `io.formats.frame.py` using a common docstring shared for `DataFrame.to_html` and `DataFrame.to_string`.
However the parameter is only used in `DataFrame.to_string` so that did not make sense to have it in a shared docstring (thus the confusion that led to the creation of issue #44304).
So I moved the documentation of `min_rows` out of the common docstring and added it only for `DataFrame.to_string` method (so it does not appear for `DataFrame.to_html` once we rebuild the documentation).